README.md

```diff
@@ -146,7 +146,7 @@ Usage:
 `{{ pm_utils.to_integer('[expression]') }}`
 
 #### to_timestamp ([source](macros/multiple_databases/to_timestamp.sql))
-This macro converts a field to a timestamp field.
+This macro converts a field to a timestamp field. For Databricks, milliseconds will automatically be parsed. You should only set the milliseconds in your datetime patern if the second fraction is different than milliseconds.
```

Comment:
Is this correct or the example above?
So this: `yyyy-MM-dd HH:mm:ss.SSS` or `yyyy-MM-dd HH:mm:ss`?

Reply:
`yyyy-MM-dd HH:mm:ss` I see what you meant, I'll fix the README.md